src/sentinel.c

```diff
@@ -3319,54 +3320,49 @@ void sentinelConfigSetCommand(client *c) {
     dictRelease(set_configs);
 }
 
-/* SENTINEL CONFIG GET <option> */
+/* SENTINEL CONFIG GET option <option option..> */
```

Comment:
`<>` denotes a variable, `[]` denotes optional.
```suggestion
/* SENTINEL CONFIG GET <option> [<option> ...] */
```

Reply:
sure, updated in the latest code